woocommerce/woocommerce-android
# [Barcode Scanner] Autofocus to the barcode scanner

Closes: #10604


### Description

There is a review from a customer that 

> The barcode reader needs to be faster and more accurate with self-adjusting focus.

We are not sure if this android or iOS. My gut feeling says that this is Android though
And this PR sets `cameraX` lib to refocus on center of the surface, BUT I on my pixel device I don't see any difference. AF works even without this settings. Maybe the issue comes on a specific device, not sure.

I was thinking to focus on the place that was tapped, but for barcode scanner it feels a bit weird - I don't see that people expect it to work like that.

### Testing instructions

* Products -> Product scanner
* Try put in center objects that a close/far away from the device
* Notice autofocus works
* Try to do the same on trunk - do you see any difference?

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/0a18e154-8c34-427b-a861-6204dfb87e1b
